src/dtx/dtx_rpc.c

```diff
@@ -37,18 +37,13 @@
 #include "dtx_internal.h"
 
 static int
-crt_proc_struct_dtx_id(crt_proc_t proc, struct dtx_id *dti)
+crt_proc_struct_dtx_id(crt_proc_t proc, struct dtx_id *p)
 {
 	int rc;
 
-	rc = crt_proc_uuid_t(proc, &dti->dti_uuid);
+	rc = crt_proc_memcpy(proc, p, sizeof(*p));
```

Comment:
Same comment as above. You are replacing a uint64 direct pointer assignment by a memcpy function. Not sure this is better.

Reply:
crt_proc_uuid_t use crt_proc_memcpy inside. So, just remove one wrapper.